CHANGELOG.md

```diff
@@ -8,6 +8,109 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [Knowledge Agents EDC](https://github.com/eclipse-tractusx/knowledge-agents-edc) | agent-plane: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents-edc/releases/tag/agent-plane-1.13.22)<br/>agent-plane-azure-vault: [1.13.22](https://github.com/eclipse-tractusx/knowledge-agents-edc/releases/tag/agent-plane-azure-vault-1.13.22) | [v1.13.22](https://github.com/eclipse-tractusx/knowledge-agents-edc/releases/tag/v1.13.22) |
+| [Managed Service Orchestrator](https://github.com/eclipse-tractusx/managed-service-orchestrator) | managed-service-orchestrator: [1.6.0](https://github.com/eclipse-tractusx/managed-service-orchestrator/releases/tag/managed-service-orchestrator-1.6.0) | [v1.5.7](https://github.com/eclipse-tractusx/managed-service-orchestrator/releases/tag/v1.5.7) |
+| [Managed Simple Data Exchanger (SDE)](https://github.com/eclipse-tractusx/managed-simple-data-exchanger) | sde: [0.1.12](https://github.com/eclipse-tractusx/managed-simple-data-exchanger/releases/tag/sde-0.1.12) | frontend: [v2.4.3](https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend/releases/tag/v2.4.3)<br/>backend: [v2.4.3](https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend/releases/tag/v2.4.3) |
+| [PCF Exchange Kit](https://eclipse-tractusx.github.io/docs-kits/category/pcf-exchange-kit) | n/a | [1.2.0](https://eclipse-tractusx.github.io/docs-kits/kits/PCF%20Exchange%20Kit/PCF%20Exchange%20Kit%20Changelog) |
+| [Policy Hub](https://github.com/eclipse-tractusx/policy-hub) | policy-hub: [1.1.0](https://github.com/eclipse-tractusx/policy-hub/releases/tag/policy-hub-1.1.0) | [v1.1.0](https://github.com/eclipse-tractusx/policy-hub/releases/tag/v1.1.0) |
+| [Portal](https://github.com/eclipse-tractusx/portal) | portal: [2.1.0](https://github.com/eclipse-tractusx/portal/releases/tag/portal-2.1.0) | registration: [v2.0.1](https://github.com/eclipse-tractusx/portal-frontend-registration/releases/tag/v2.0.1)<br/>frontend: [v2.1.0](https://github.com/eclipse-tractusx/portal-frontend/releases/tag/v2.1.0)<br/>backend: [v2.1.0](https://github.com/eclipse-tractusx/portal-backend/releases/tag/v2.1.0)<br/>assets: [v2.1.0](https://github.com/eclipse-tractusx/portal-assets/releases/tag/v2.1.0) |
```

Comment:
@evegufy: is the app registration's version incorrect? 2.10 is set in corresponding chart, but here it's 2.0.1

If 2.1.0 is correct, please correct as follows
```suggestion
| [Portal](https://github.com/eclipse-tractusx/portal) | portal: [2.1.0](https://github.com/eclipse-tractusx/portal/releases/tag/portal-2.1.0) | registration: [v2.1.0](https://github.com/eclipse-tractusx/portal-frontend-registration/releases/tag/v2.1.0)<br/>frontend: [v2.1.0](https://github.com/eclipse-tractusx/portal-frontend/releases/tag/v2.1.0)<br/>backend: [v2.1.0](https://github.com/eclipse-tractusx/portal-backend/releases/tag/v2.1.0)<br/>assets: [v2.1.0](https://github.com/eclipse-tractusx/portal-assets/releases/tag/v2.1.0) |
```

Reply:
2.0.1 is correct for the registration frontend, see [here](https://github.com/eclipse-tractusx/portal/blob/portal-2.1.0/charts/portal/values.yaml#L144)
I also already mentioned it here https://github.com/eclipse-tractusx/tractus-x-release/pull/34#issuecomment-2265111269